Failing to generate API documentation using `make gendocs`

**Do you want to request a *feature* or report a *bug*?**
*bug*

**What is the current behavior?**
I am failing to generate API documentation .

After running `make gendocs` I get the following output:
```
node website.js
/var/www/tmp/mongoose/docs/source/api.js:166
        ctx.anchorId = `${ctx.name.toLowerCase()}_${ctx.name}`;
                                   ^

TypeError: Cannot read property 'toLowerCase' of undefined
    at parse (/var/www/tmp/mongoose/docs/source/api.js:166:36)
    at Object.<anonymous> (/var/www/tmp/mongoose/docs/source/api.js:54:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/var/www/tmp/mongoose/docs/splitApiDocs.js:3:13)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/var/www/tmp/mongoose/website.js:88:1)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at node:internal/main/run_main_module:17:47
make: *** [Makefile:21: gendocs] Error 1
```

**If the current behavior is a bug, please provide the steps to reproduce.**
Clone master branch and run:
```
npm install
make docclean
make gendocs
```

**What is the expected behavior?**
`make gendocs` should execute without throwing error described above.

**What are the versions of Node.js, Mongoose and MongoDB you are using? Note that "latest" is not a version.**
- Node.js v16.6.1
- Mongoose v6.1.1

--- a/lib/query.js
+++ b/lib/query.js
@@ -5213,7 +5213,7 @@ Query.prototype.tailable = function(val, opts) {
 
   if (opts && typeof opts === 'object') {
     for (const key of Object.keys(opts)) {
-      if (key === 'awaitData' || key === 'awaitdata' /* backwards compat, see gh-10875 */) {
+      if (key === 'awaitData' || key === 'awaitdata') { // backwards compat, see gh-10875
         // For backwards compatibility
         this.options['awaitData'] = !!opts[key];
       } else {

